components/HackneyDS/FormGroup/index.js

```diff
@@ -1,20 +1,21 @@
 import React from 'react';
 import { HorizontalSeparator } from '../index';
 
-export default function FormGroup({ children = [], title, hint, error }) {
+export default function FormGroup ({ children = [], label, hint, error, className = '' }) {
+  const outerClassName = className ? ` ${className}` : '';
   const nodeList = Array.isArray(children) ? children : [children];
   const errorClassList = error ? ' govuk-form-group--error' : '';
 
   return (
-    <div className={`govuk-form-group lbh-form-group${errorClassList}`}>
+    <div className={`govuk-form-group lbh-form-group${errorClassList}${outerClassName}`}>
       <fieldset className="govuk-fieldset lbh-fieldset">
-        <legend className="govuk-fieldset__legend">{title}</legend>
+        <legend className="govuk-fieldset__legend">{label}</legend>
```

Comment:
- This change affects the RadioGroup storybook (rename prop in storybook to label).
- Change <legend> to <Label> component